### PR TITLE
Add order status to order response

### LINKED ISSUE #56: Add order state to order response
Add the Grossman order status to the order response.

Convert the number to some nice text, as below (but we'll snakecase it):

```
0	No Items
1	Ordered
2	Invoiced
3	Deleted
```

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Add the order status and map to a friendly symbol in the Order
and Order::Reference models.

Closes #56

### CODE REVIEW

File: db/migrate/20160901233059_add_in_ord_status_cd_to_orders.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class AddInOrdStatusCdToOrders < ActiveRecord::Migration
+  return if Order
```
Comment (wopr42) [inline db/migrate/20160901233059_add_in_ord_status_cd_to_orders.rb]: Why is this here instead of inside the change method?

**Comment by wopr42** [comment]
Just a question out of curiosity, looks good to me.

File: db/migrate/20160901233059_add_in_ord_status_cd_to_orders.rb (hunk 1 above)
Comment (josephbridgwaterrowe) [inline db/migrate/20160901233059_add_in_ord_status_cd_to_orders.rb]: To be honest I'm not sure, I suppose it would work in either place.

I could try moving it, although it does work where it is.

Comment (josephbridgwaterrowe) [inline db/migrate/20160901233059_add_in_ord_status_cd_to_orders.rb]: @wopr42, moved that check in all migrations... makes more sense being in the method.
